src/content/docs/alerts-applied-intelligence/new-relic-alerts/learn-alerts/alerts-ai-transition-guide-2022.mdx

```diff
@@ -0,0 +1,105 @@
+
+  **Learn more:**
+
+  * [Faceted baseline conditions documentation](/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/other-condition-types/create-baseline-alert-conditions/#baseline-facets)
+  * [Explorers Hub](https://discuss.newrelic.com/t/nrql-outlier-alert-conditions-end-of-life/164167)
+
+  </Collapser>
+
+  <Collapser
+    id="query-thresholds"
+    title="May 2022: Sliding window aggregations to replace sum of query results thresholds."
+  >
+
+  **Reason for EOL:** This has not been an ideal way to address issues caused by sporadic signals. It lacks the ability to smooth out volatile signals or efficiently deal with gaps in signal.
+
+  **Replacement:** Development for full support for sliding window aggregation in NRQL conditions is underway.
+
+  **Learn more:**
+  
+  * Additional documentation will be provided.
+  * [Explorers Hub](https://discuss.newrelic.com/t/big-news-for-alerts-and-applied-intelligence/161436)
+
+  </Collapser>
+
+  <Collapser
+    id="incident-response"
+    title="2nd half of 2022: Native experience in New Relic One to replace classic incident response workflow."
+  >
+
+  **Reason for EOL:** The classic incident response workflow involves the **Incident** and **Violation details** pages in the UI. It also uses APIs for incidents and violations.
+
+  **Replacement:** We will replace the current experience by rolling out improved incident response experiences with a new, native UI experience in New Relic One.
+
+  **Learn more:**
+
+  * Additional documentation will be provided.
+  * [Explorers Hub](https://discuss.newrelic.com/t/big-news-for-alerts-and-applied-intelligence/161436)
+
+  </Collapser>
+
+  <Collapser
+    id="notifications"
+    title="2nd half of 2022: Workflows and destinations to replace notification channels and relationship to policies."
+  >
+
+  **Reason for EOL:** Current notification channels are rigid and cannot be customized. They also require restrictive policy-based alert conditions that focus on incident creation.
+
+  **Replacement:** Notification channels will be replaced by workflows and destinations, a new way to manage issue-driven notifications. This solution is more flexible, and it enables notifications for all severity levels.
+
+  **Learn more:**
+
+  * Additional documentation will be provided.
+  * [Explorers Hub](https://discuss.newrelic.com/t/big-news-for-alerts-and-applied-intelligence/161436)
+
+  </Collapser>
+
+  <Collapser
+    id="condition-types"
+    title="2nd half of 2022: NRQL conditions to replace all non-NRQL alert condition types."
+  >
+
+  **Reason for EOL:** A complex user experience and data platform has prevented advances in detection capabilities.
```

Comment:
I feel like this needs a little more explanation and links to explain it well. Something like: 

"Currently [NRQL alert conditions](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions) are one type of alert condition. But our many types of alert conditions have resulted in a complex user experience and the lack of alignment has prevented advances in detection capabilities...." 

Something to make it a little more clear what's tied to the 'complex user experience'. 
at least that's what I'm grokking from it.

Reply:
I also kind of wonder if there are other takeaways to this, like 'Consider only using NRQL alert conditions from here on out when creating new ones to be most aligned with this change'. Maybe that's mentioned in the forum post but theoretically if it important enough could be worth promoting here prominently.

Reply:
I did just see that there are recommendations of what to do (like one above) in the forum post, and I wonder if we should promote that more heavily at top of this doc or within each item. Because I can imagine someone reading this doc and not getting what I see as the main point; that they should be understanding the future impacts and doing some specific recommendations (e.g., switching to NRQL alert conditions) to prep for these EOLs. I can imagine them not getting that there are big takeaways in the forum post, in other words.

Reply:
This is a good point. We really want them to use the Explorers Hub post as a communication point, esp over time, but it won't hurt to put more detail here. Thanks for the suggestions!